### ISSUE TITLE
IconViews used in buttons should be marked as decorative since they lack accessible labels

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#17554]
## 📝 Provide detailed reproduction steps (if any)

Even though toolbar buttons have their own labels, the icons used inside don't, and accessing them individually, e.g. using the "virtual cursor" feature offered by assistive technologies presents an accessibility issue. 

### ❓ Possible solution

Icons that are used as a decoration should come with an `aria-hidden` attribute.

### Possible questions

Should all `IconView` get `aria-hidden` or just those used in buttons/menus/dropdowns/lists? Frankly, I don't remember any place using `IconView` for anything other than decoration but who knows (3rd-party). It might be a BC so it needs consideration.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by DawidKossowski** [comment]
Context: https://app.slack.com/client/T03UQQ7LL/CCGP16N4R
